sljit_src/sljitNativeLOONGARCH_64.c

```diff
@@ -3618,14 +3616,42 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_atomic_load(struct sljit_compiler
 	CHECK_ERROR();
 	CHECK(check_sljit_emit_atomic_load(compiler, op, dst_reg, mem_reg));
 
+	if (!(LOONGARCH_CFG2_LAMCAS & get_cpu_features(GET_CFG2))) {
+		switch (GET_OPCODE(op)) {
+		case SLJIT_MOV:
+		case SLJIT_MOV_P:
+			ins = LL_D;
```

Comment:
I don't remember the current status, but I suspect LL_D is not available for 32 bit systems. Do we only support 64 bit?

Reply:
By now, almost every LoongArch chip are 64-bit, AFAIK, only 1c series mcu are 32-bit, so I think it's safe to support 64-bit only. If there were 32-bit chips come into market, we could follow the way RISC-V did, split this into sljitNativeLOONGARCH_common.c and so-on.